owncloudApp/src/main/java/com/owncloud/android/ui/activity/ReceiveExternalFilesActivity.java

```diff
@@ -613,7 +613,9 @@ private void startSyncFolderOperation(OCFile folder) {
 
         mSyncInProgress = true;
 
-        mReceiveExternalFilesViewModel.refreshFolderUseCase(folder);
+        if (mReceiveExternalFilesViewModel != null) {
+            mReceiveExternalFilesViewModel.refreshFolderUseCase(folder);
+        }
```

Comment:
Not sure if the NPE refers to this line or something inside the `refreshFolderUseCase` method... `mReceiveExternalFilesViewModel` shouldn't be `null` because it's obtained by Koin dependency injection in a previous executed method 🤔

Reply:
Yes, true, but with the information we have in the trace we do not know exactly what is causing the null. Since we were also unable to reproduce the error, I would opt for this approach.